### ISSUE TITLE
Translation

### ISSUE THREAD

**Issue by nortigo** [jazzband/django-defender#178]
Hi,

I noticed there is no translation. Is there any reasons for that?

**Comment by kencochrane** [comment]
No, not really. We just haven't had the time. If someone wants to contribute, we are more than willing to accept the PR.
